Server/mods/deathmatch/logic/CElement.h

```diff
@@ -91,11 +91,11 @@ class CElement
     virtual CElement* Clone(bool* bAddEntity, CResource* pResource) { return nullptr; }
     bool              IsCloneable();
 
-    bool         IsBeingDeleted() { return m_bIsBeingDeleted; };
-    void         SetIsBeingDeleted(bool bBeingDeleted) { m_bIsBeingDeleted = bBeingDeleted; };
+    bool         IsBeingDeleted() { return m_bIsBeingDeleted; }
+    void         SetIsBeingDeleted(bool bBeingDeleted) { m_bIsBeingDeleted = bBeingDeleted; }
```

Comment:
const method qualifier?

Reply:
For other PR, I was in CElement only to add `const` qualifier to the `GetID` method